ssl/quic/quic_impl.c

```diff
@@ -3912,6 +3912,59 @@ int ossl_quic_get_key_update_type(const SSL *s)
     return SSL_KEY_UPDATE_NONE;
 }
 
+/*
+ * QUIC Front-End I/O API: Listeners
+ * =================================
+ */
+
+SSL *ossl_quic_new_listener(SSL_CTX *ctx, uint64_t flags)
+{
+    QUIC_LISTENER *ql = NULL;
+    QUIC_ENGINE_ARGS engine_args = {0};
+    QUIC_PORT_ARGS port_args = {0};
+
+#if defined(OPENSSL_THREADS)
+    if ((ql->mutex = ossl_crypto_mutex_new()) == NULL) {
```

Comment:
I'm going to assume this has been fixed by a subsequent commit....we are derefing `ql` before it has been assigned.